src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -94,8 +102,8 @@ export class CreateScenariosComponent implements OnInit {
         (control: AbstractControl) =>
           scenarioNameMustBeNew(control, this.existingScenarioNames),
       ]),
-      priorities: this.prioritiesComponent.createForm(),
-      constrains: this.constraintsPanelComponent.createForm(),
+      priorities: this.prioritiesComponent?.createForm() || '',
```

Comment:
I don't think we should add optional chaining here, we wanna be sure we are creating the form, otherwise something else is wrong in the code